SiennaOpenAPIModels.jl/src/sienna_to_json/static_injections.jl

```diff
@@ -192,3 +192,38 @@ function psy2openapi(hydro::PSY.HydroPumpedStorage, ids::IDGenerator)
         dynamic_injector=getid!(ids, hydro.dynamic_injector),
     )
 end
+
+function psy2openapi(energy_res::PSY.EnergyReservoirStorage, ids::IDGenerator)
+    EnergyReservoirStorage(
+        id = getid!(ids, energy_res),
+        name = energy_res.name,
+        available = energy_res.available,
+        bus = getid!(ids, energy_res.bus),
+        prime_mover_type = string(energy_res.prime_mover_type),
+        storage_technology_type = string(energy_res.storage_technology_type),
+        storage_capacity = energy_res.storage_capacity,
+        storage_level_limits = get_min_max(
+            scale(energy_res.storage_level_limits, PSY.get_base_power(energy_res)),
```

Comment:
Looking at the list of converted entries in PowerSystems.jl, storage_capacity should be converted to natural units, but storage_level_limits doesn't need to be..